src/assets/scss/_variables.scss

```diff
@@ -28,4 +28,4 @@ $primary-light: #474787;
 $accent-dark: #cd6133;
 $accent-light: #ff793f;
 $other-dark: #d1ccc0;
-$other-light: #ffffff;
\ No newline at end of file
+$other-light: #ffffff;
```

Comment:
Must have an empty line in the file end

Reply:
![image](https://user-images.githubusercontent.com/23269934/162721505-e40dedd2-6c5f-4a34-ba34-cade7ba24596.png)